app/rest/openshift/src/main/java/io/syndesis/openshift/OpenShiftServiceImpl.java

```diff
@@ -193,7 +179,9 @@ private void ensureDeploymentConfig(String name, DeploymentData deploymentData)
             .withNewMetadata().addToLabels("integration", name).endMetadata()
             .withNewSpec()
             .addNewContainer()
-            .withImage(" ").withImagePullPolicy("Always").withName(name)
+                .withImage("")
```

Comment:
At some point we had an issue with an 0 length string for the image name. Not sure anynmore where. Have you verified that this works ?